mon-pix/translations/fr.json

```
@@ -479,6 +488,10 @@
       "succeed": "Votre mot de passe a été modifié avec succès."
     },
 
+    "qcu-proposals": {
+      "answer": "La réponse à la question est : "
+    },
+
```

Il pourrait y avoir aussi `qroc-proposal`, `qrocm-proposal`, etc. Comme ils seront ordonné par ordre alphabétique, et pour éviter de les éparpiller, je propose de les regrouper:
```suggestion
"proposals": {
  "answer": {
    "qcu": "La réponse à la question est : "
  }
}

```